roles/openshift_aws/tasks/dns_route53.yml

```diff
@@ -0,0 +1,23 @@
+---
+- set_fact:
+    l_dns_zones: "{{ l_dns_zones | default([]) }} + [ '{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['private_zone'] }}' ]"
+  with_items: "{{ openshift_aws_dns_records }}"
+  loop_control:
+    loop_var: l_openshift_aws_dns_element
+
+- name: creating route53 zone(s)
+  route53_zone:
+    comment: "{{ openshift_aws_dns_zone }}"
+    state: present
+    vpc_id: "{{ (l_openshift_aws_route53_scheme == 'True') | ternary('', vpcout.vpcs.0.id) }}"
```

Comment:
vpc_id: "{{ item }}"

Reply:
Also, where are we getting vpcout? I don't see that in this play.

Reply:
It's from an imported task.  See playbooks/aws/openshift-cluster/roles/openshift_aws/tasks/dns.yaml#L2